tests/Integration/Behaviour/Features/Scenario/Discount/add_discount.feature

```diff
@@ -0,0 +1,43 @@
+# ./vendor/bin/behat -c tests/Integration/Behaviour/behat.yml -s discount --tags add-discount
+@restore-all-tables-before-feature
+@add-discount
+
+Feature: Add discount
+  PrestaShop allows BO users to create discounts
+  As a BO user
+  I must be able to create discounts
+
+  Background:
+    Given shop "shop1" with name "test_shop" exists
+    Given there is a currency named "usd" with iso code "USD" and exchange rate of 0.92
+    Given there is a currency named "chf" with iso code "CHF" and exchange rate of 1.25
+    Given currency "usd" is the default one
+    And language with iso code "en" is the default one
+
+  Scenario: Create a discount with free shipping
+    When I create a free shipping discount "discount_1" with following properties:
```

Comment:
It's cool that you already handled some additional optional fields, but can you please add a scenario with the bare minimum attributes (so only type) that checks that it correctly creates the discount?